src/reducers/issue.js

```diff
@@ -26,18 +26,14 @@ export function issues(state = initialState, action) {
         error: action.error,
         loading: false,
       })
-    case types.REQUEST_ISSUE_BY_ID:
-      return Object.assign({}, state, {
-        data: action.data || [],
-        error: false,
-        loading: true,
-      })
-    case types.REQUEST_ISSUE_BY_ID_SUCCESS:
-      return Object.assign({}, state, {
-        data: action.data || [],
-        error: false,
-        loading: true,
-      })
+    case types.UPDATE_VOTE_COUNT:
+      let newState = Object.assign({}, state);
+      newState.data.map(issue => {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

It creates and returns a new array, but it's not being returned anywhere; if it's working, it's working in spite of the code